### ISSUE TITLE
Application Accessing Deleted Blob Storage Documents

### ISSUE THREAD

**Issue by usmantariq-odl** [Azure-Samples/aisearch-openai-rag-audio#61]
I am encountering an issue where the application continues to access and provide answers referencing documents that have already been deleted from the Blob Storage. Despite removing these documents, it seems the system is still using them as part of its data source.

@pamelafox , Could you please investigate why this is happening and provide guidance or a resolution to ensure deleted documents are no longer accessible or referenced by the application?

**Comment by pamelafox** [comment]
@usmantariq-odl Are you using the integrated vectorization approach, the default for this repo, or are you bringing your own index? Every indexing approach will have different ways they handle blob processing.

If you're using integrated vectorization, have you re-run the indexer after removing them? It's not currently run on a schedule.

**Comment by usmantariq-odl** [comment]
@pamelafox , Yes I'm using the Integrated vectorization approach. I recreated the indexer  by deleting it and deploying app again and it worked fine then. But i couldn't find a way to just clean the indexer or reset it directly from Azure, if you know the best approach. Please let me know.

Kind regards,
Usman Tariq

**Comment by usmantariq-odl** [comment]
Hi @pamelafox,

After deleting a document from Blob Storage, the system still references it when I ask questions. To resolve this, I reset and reran the indexer, but the deleted document remains part of the data source. The only way I was able to fully remove it was by redeploying the application.

Since we've discussed this before, could you suggest a solution to ensure deleted documents are properly removed without requiring a full redeployment? The reset and re-run approach doesn’t seem to work for me.
